zerver/openapi/zulip.yaml

```diff
@@ -15188,6 +15193,11 @@ paths:
 
                           Important for clients containing UI where one can browse channels to subscribe
                           to.
+
+                          **Changes**: Prior to Zulip 10.0 (feature level 344), if a user was
+                          administrator of a channel that they never subscribed to, but not
```

Comment:
It might be better to say "if a user was in `can_manage_channel_group` for a channel ..." just to be more precise about the condition in question.